### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by chengjiaoyang** [sequelize/sequelize-auto#462]
hi ，i using Mac computer command line to input “sequelize-auto -h 127.0.0.1 -d test -u root -x 123 -p 3306” error
env: node\r: No such file or directory

**Comment by steveschmitt** [comment]
Try using
```
node_modules/.bin/sequelize-auto -h 127.0.0.1 -d test -u root -x 123 -p 3306
```

**Comment by papb** [comment]
This `node\r` error is usually caused by having a shebang such as `#!/usr/bin/env node` in the first line of a file but with CRLF line endings, which will cause some parsers to see the last part as `node\r` (since they might be expecting LF endings only).

**Comment by chengjiaoyang** [comment]
hello  it work for me
:set ff=unix  thank u~

**Comment by steveschmitt** [comment]
Fixed in 0.7.9
